RELEASE-NOTES.txt

```diff
@@ -1,6 +1,7 @@
 4.3
 -----
 - Products: now the Product details can be edited and saved outside Products tab (e.g. from Order details or Top Performers).
+- Products: we improved our VoiceOver support in Product Price
```

Comment:
nit: might be worth mentioning the "settings" like `Products: we improved our VoiceOver support in Product Price settings` so that it's clear it's not the price on the product details screen?